arrow/constants.py

```diff
@@ -1,3 +1,8 @@
+"""
+Constants used internally in arrow..
+
+"""
```

Comment:
This can probably made into a one line docstring.

Also you got a double period.